build/docker/docker-compose.yml

```diff
@@ -1,5 +1,6 @@
 mariadb:
     image: mariadb:latest
+    command: mysqld --character-set-server=utf8mb4 --collation-server=utf8mb4_unicode_ci --sql_mode=""
```

Comment:
@tarrsalah 
you can not use utf8mb4 with dolibarr !

Reply:
@hregis Thanks for comment, didn't know about the utf8mb4 support problem in dolibarr, show I undo my second commit before the merge?

Reply:
@tarrsalah This is not a problem in Dolibarr but into mysql/mariadb themself.
utf8mb4 need 4 chars for each char, but the limit of 767 char per index was not increased. So it means a field with varchar(255) will create an error if you have an index on it.
As long as mysql/mariadb has not increased this, it is not possible to recommand utf8mb4 because it means you must choose between using utf8 + index or utf8mb4 but removing index on all char fields that have more than 188 chars. Of course, we prefer first choice when we know that varchar can contains 255 and not only 188 chars.